### DISCUSSION TITLE
"Export as gltf 2.0 scene" but for 2D? Export scene as SVG?

### DISCUSSION THREAD

**Discussion by maksutheam** [godotengine/godot-proposals#5837] [2D]
So, lets say that I had prototyped a layout for a 3D scene with primitive meshes and GSC and wanted to make it pretty with my favorite 3D program, like Blender. I could bring the scene there by exporting it as a gltf 2.0. When I'm done, I can just open the same file in Godot. It's simple and it works!

Now, lets say that I had prototyped a layout for a 2D scene with primitive shapes and polygons and wanted to make it pretty with my favorite 2D program, like Krita. What can I do? I guess I could eyeball my drawings, but if they're wrong, which they usually are on the first try, I would have to go back and forth between the programs to get it to look right, which is rather tedious.

I had this problem when I tried making a game for a gamejam, and I was wondering if it could be possible to export 2D scenes to something that drawing programs could read. Maybe SVG? Or maybe my problem is too specific to even justify a built-in solution? I'm not sure, which is why I made this discussion.

**Comment by Calinou** [comment]
The SVG specification is extremely complex, so writing this exporter will also be extremely complex. There's no simpler 2D standard vector format out there, unfortunately.

That said, it sounds like an interesting idea to prototype with an add-on.
